src/WattleScript.Interpreter/Errors/ScriptRuntimeException.cs

```diff
@@ -501,6 +501,40 @@ public static ScriptRuntimeException AccessInstanceMemberOnStatics(IUserDataDesc
 		{
 			return new ScriptRuntimeException("attempt to access instance member {0}.{1} from a static userdata", typeDescr.Name, desc.Name);
 		}
+		
+		/// <summary>
+		/// Creates a ScriptRuntimeException with a predefined error message specifying that
+		/// an attempt to assign an invalid value to a new range was made
+		/// </summary>
+		/// <param name="property">To/From</param>
+		/// <param name="type">Unexpected type casted to string</param>
+		/// <returns></returns>
+		public static ScriptRuntimeException NewRangeBadValue(string property, string type)
+		{
+			return new ScriptRuntimeException("bad value '{0}' for new range (number expected, got {1})", property, type);
+		}
+		
+		/// <summary>
+		/// Creates a ScriptRuntimeException with a predefined error message specifying that
+		/// an attempt to assign an invalid value to a existing range was made
+		/// </summary>
+		/// <param name="type">Unexpected type casted to string</param>
+		/// <returns></returns>
+		public static ScriptRuntimeException ExistingRangeBadValueAssigned(string type)
+		{
+			return new ScriptRuntimeException("bad value assinged to a range property (number expected, got {0})", type);
```

Comment:
Typo: assigned. 

More concise `bad value for range '{0}' (number expected, got {1})`. This can be used for both property and range construction. "new range" is redundant

Reply:
Message changed as suggested